programs/cleos/main.cpp

```diff
@@ -482,6 +482,20 @@ void print_action( const fc::variant& at ) {
    */
    if( args.size() > 100 ) args = args.substr(0,100) + "...";
    cout << "#" << std::setw(14) << right << receiver << " <= " << std::setw(28) << std::left << (code +"::" + func) << " " << args << "\n";
+
+   std::string return_value, return_value_prefix{"return value: "};
+   try {
+      return_value = fc::json::to_string(at["return_value_data"], fc::time_point::maximum());
+   } catch (...) {
```

Comment:
What exceptions are you catching?  Can you specify the exception before catching the default exception?
Or can you find "return_value_data" instead of exception?